### PR TITLE
Updated the docs for generating local manifests

### PR DESCRIPTION
**Pull request by Himangini**
### Description
Part of https://github.com/weaveworks/pctl/pull/289
Updated docs to refect changes for the installation directory name

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md` (diagrams, usage, roadmap etc), and anything in `examples/`)
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

**Review by aclevername — CHANGES_REQUESTED** [review]
do we need to change any other parts of the docs now `--name` is mandatory?

**Comment by Himangini** [comment]
> do we need to change any other parts of the docs now `--name` is mandatory?

yup, added changes everywhere to reflect the name flag required with the `add` command

File: userdocs/profiles.dev/docs/installer-docs/generate-local-manifests.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ related flags.
 For example, using a profile URL:
 ```yaml
 pctl add \
+  --name pctl-profile \
```
Comment (aclevername) [inline userdocs/profiles.dev/docs/installer-docs/generate-local-manifests.md:14]:
```suggestion
  --name <installation-name> \
```

keeping it consistent with the other arguments

File: userdocs/profiles.dev/docs/installer-docs/generate-local-manifests.md [hunk 2]
```diff
@@ -28,23 +29,24 @@ Example generating from a profile listed in a catalog:
 
 ```yaml
 pctl add \
+  --name pctl-profile \
```
Comment (aclevername) [inline userdocs/profiles.dev/docs/installer-docs/generate-local-manifests.md:32]:
```suggestion
  --name <installation-name> \
```

File: userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md [hunk 3]
```diff
@@ -107,6 +109,7 @@ and destination of your PR. For example, the following command will:
 
 ```bash
 pctl add \
+  --name pctl-profile \
```
Comment (aclevername) [inline userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md:112]:
```suggestion
  --name <profile installation name> \
```

Comment (Himangini) [reply userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md:112]: the example here has dummy names of the flags like `--pr-repo drwho/thirteen` so i am keeping the name suggestion here

Comment (aclevername) [reply userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md:112]: can we have a more meaningful example then? `nginx-profile` or something like that

Comment (Himangini) [reply userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md:112]: i think this is fine
```
pctl add \
  --name pctl-profile \
  --profile-repo-url https://github.com/weaveworks/nginx-profile \
  --create-pr \
  --pr-repo drwho/thirteen \
  --out ethel \
  --namespace two-sugars-please \
  --pr-branch add-aunt-ethels-profile \
  --pr-remote jammy-dodger \
  --pr-base tea-time
  ```

Comment (aclevername) [reply userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md:112]: `pctl-profile` doesn't really mean anything though, its just the name of the two tools :shrug:

**Review by aclevername — APPROVED** [review]

File: userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md (hunk 3 above)
Comment (Himangini) [reply userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md:112]: the rest of the example doesn't have meaningful names either 😂  let's not be pedantic 😄

Comment (aclevername) [reply userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md:112]: I sort of dislike that the rest aren't useful :joy: (although `--profile-repo-url` is a real example) the docs should help users make better decisions, e.g. good branch/installation namings

**Comment by aclevername** [comment]
we haven't released the pctl changes for this yet :see_no_evil: but hopefully should be live soon anyway https://github.com/weaveworks/pctl/pull/291

**Comment by Himangini** [comment]
> we haven't released the pctl changes for this yet 🙈 but hopefully should be live soon anyway [weaveworks/pctl#291](https://github.com/weaveworks/pctl/pull/291)

oops yeah, you are right 🙈 , although the name flag was always there so doesn't affect the user much I guess 😓  🤷🏻‍♀️

**Pull request merged**